README.md

```diff
@@ -1,13 +1,21 @@
-## Studio Library
+<p align="center">
```

Comment:
Would it be possible to have the icon to the left of the title and then align the title to the left?